tests/MongoDB.Driver.Tests/JsonDrivenTests/JsonDrivenSessionTest.cs

```diff
@@ -0,0 +1,48 @@
+﻿/* Copyright 2018-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System.Collections.Generic;
+using MongoDB.Bson;
+using MongoDB.Bson.TestHelpers.JsonDrivenTests;
+
+namespace MongoDB.Driver.Tests.JsonDrivenTests
+{
+    public abstract class JsonDrivenSessionTest : JsonDrivenCommandTest
```

Comment:
This is the base class for all tests where _session is the receiver of the method call.